### ISSUE TITLE
Can I use MZmine process the polarity swithing data? And how?

### ISSUE THREAD

**Issue by mliu2011** [mzmine/mzmine2#886]
### Basic information

* My operating system: windows 7
* My MZmine version: 2.53

### What happened

I have some mass spectrometry data collected by polarity switching. Can MZmine process these data?

**Comment by photocyte** [comment]
Yes. If you use the Raw data methods->Filtering->Crop filter module to make virtual raw files with just the positive or negative data, things will probably work better.  Hit the "Set filters" button in the Crop filter module to set filtering to just positive or negative mode.

**Comment by mliu2011** [comment]
Hello,

Thanks very much for your reply!

I actually tested the cropping to filter out just the negative data, but I notice that some of the files the cropping (filtering) can not get all the data points. For example, one of the metabolites, XYLITOL, shows 1E6 intensity in Xcalibur, but only shows 1E4 intensity in MZmine after cropping. I have totally 6 samples (raw files), 2 out of 6 files have problems (intensity is reduced 100 times). In the setting for filters, I leave everything blank, and just chose "negative". I thought that would give me the least rigid filtering. Can you please let me know how to fix this issue?



Thanks,



[signature_1324802923]

Min Liu, Ph. D.
Staff Scientist, Proteomics and Metabolomics Core
Moffitt Cancer Center

12902 Magnolia Drive, Tampa, FL 33612, MRC 1296, 813-745-1192 | email: min.liu@moffitt.org<mailto:M@moffitt.org>







________________________________
